packages/repeater/src/register.ts

```diff
@@ -66,7 +70,9 @@ container.register(RMQEventBusConfig, {
     (childContainer: DependencyContainer) => ({
       exchange: 'EventBus',
       appQueue: 'app',
-      clientQueue: `agent:${childContainer.resolve(RepeaterId)}`
+      clientQueue: `agent:${
+        (childContainer.resolve(RepeaterBus) as RepeaterBus).repeaterId
```

Comment:
I don't understand this logic. What does `repeaterId` mean in terms of the Bus?

Reply:
reverted